sdk/keyvault/azure-keyvault-administration/tests/perfstress_tests/README.md

```diff
@@ -0,0 +1,48 @@
+# azure-keyvault-administration Performance Tests
+
+In order to run the performance tests, the `azure-devtools` package must be installed. This is done as part of the
+`dev_requirements` install. Start by creating a new virtual environment for your perf tests. This will need to be a
+Python 3 environment, preferably >=3.7.
+
+### Setup for test resources
+
+The following environment variables will need to be set for the tests to access the live resources:
+
+```
+AZURE_TENANT_ID=<tenant ID of testing service principal>
+AZURE_CLIENT_ID=<client ID of testing service principal>
+AZURE_CLIENT_SECRET=<client secret of testing service principal>
+AZURE_KEYVAULT_URL=<URL of the testing key vault>
```

Comment:
Just to clear up ambiguity, we should probably use `AZURE_MANAGEDHSM_URL` instead of `AZURE_KEYVAULT_URL` in these tests